client-js/app/scripts/app.ts

```diff
@@ -247,17 +257,102 @@ window.addEventListener('WebComponentsReady', () => {
     }
   }
 
+  class DatavaultLiveActivity implements Activity {
```

Comment:
I'm thinking of activities being in 1-1 correspondence with URLs or locations in the app. The DatavaultActivity shows the contents of a particular directory; the DatavaultLiveActivity shows a live-updating view of the most recent datasets in a directory. Since these do substantially different things, I made them separate activity classes. I'd be happy to change the names if you think there's a way to more clearly indicate what is going on. Also, I'll add some docstrings since that also helps to communicate the intent.